webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -3003,7 +3002,7 @@ class StringFormatter private (
    */
   def validateUUIDOfResourceIRI(iri: SmartIri): Unit =
     if (iri.isKnoraResourceIri && hasUUIDLength(iri.toString.split("/").last) && !isUUIDVersion4Or5(iri.toString)) {
-      throw BadRequestException(UUID_INVALID_ERROR)
+      throw BadRequestException(UuidInvalid)
```

Comment:
Here you use a different import style than below (importing the error directly, as opposed to importing the package and then using `IriErrorMessages.UuidInvalid`).

Reply:
Not at all, just a consistency thing

Reply:
OK, I prefer to have it like above, only imported property wo part of package path, but somehow vscode like the other way around...